styleguide/screens/Patterns/Buttons/Buttons.css

```diff
@@ -11,3 +11,8 @@
   vertical-align: middle;
   display: inline-block;
 }
+
+.componentTitle {
+	padding-top: var(--size-lg-ii);
```

Comment:
New CSS rules should follow the guidelines, which means setting indentation to two spaces. Should also move the rule to `Styleguide.css` and import that as needed